test/parallel/test-http-insecure-parser.js

```diff
@@ -0,0 +1,42 @@
+// Flags: --insecure-http-parser
+
+'use strict';
+require('../common');
+const assert = require('assert');
+const http = require('http');
+const net = require('net');
+
+const server = http.createServer(function(req, res) {
+  assert.strictEqual(req.headers['content-type'], 'text/te\bt');
+  req.pipe(res);
+});
+
+server.listen(0, test);
+
+server.once('clientError', (err, sock) => {
+  console.log('Unexpected clientError:', err);
+  sock.destroy();
```

Comment:
Shouldn’t this fail the test?

Reply:
It does, `assert.strictEqual(head, 'HTTP/1.1 200 OK');` will fail if the socket is destroyed.